pkgs/by-name/se/servo/package.nix

```diff
@@ -0,0 +1,130 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  python3,
+  fontconfig,
+  freetype,
+  libunwind,
+  xorg,
+  gst_all_1,
+  taplo,
+  cargo-deny,
+  llvmPackages_14,
+  udev,
+  cmake,
+  dbus,
+  gcc,
+  git,
+  pkg-config,
+  which,
+  llvm_14,
+  perl,
+  yasm,
+  m4,
+  libxkbcommon,
+  zlib,
+  vulkan-loader,
+  wayland,
+  libGL,
+  gnumake42,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "servo";
+  version = "unstable-2024-08-09";
```

Comment:
Please read https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-naming. The commit message also has to be fixed.

Reply:
This is still a draft and will be cleaned up before submitted. Opened as a draft for tracking purposes only at this stage.